exporters/variables-scss/src/content/formatters/cssFormatter.ts

```diff
@@ -0,0 +1,25 @@
+export const formatCSS = (css: string): string => {
```

Comment:
I have a hard time understanding what is the purpose of this function.

I am wondering if it would make sense to add comments with examples. What do you think?

Reply:
Better to add the test.

Reply:
test added